server/routes/chat_routes.ts

```diff
@@ -290,6 +290,14 @@ export function registerChatRoutes(router: IRouter, routeOptions: RoutesOptions)
         const getResponse = await storageService.getConversations(request.query);
         return response.ok({ body: getResponse });
       } catch (error) {
+        const errorBody = error?.meta?.body?.error;
+        // Ignore index not found exception for .plugins-ml-memory-meta index
+        if (
+          errorBody?.type === 'index_not_found_exception' &&
+          errorBody?.reason.includes('.plugins-ml-memory-meta')
```

Comment:
we can use status code 404? hardcode index name here is not a good practice, index name may changes in future.

Reply:
Sure, it can be changed to a 404 status code to determine if we need to return an empty result. The reason for using `plugins-ml-memory-meta` here is that I'm not sure if any other 404 errors will be thrown here. What do you think about this?

Reply:
I think use status code 404 should be safe, considering support aoss or multi tenancy, we don't have the assumption that the backend storage is OpenSearch index.

Reply:
Updated, could you help me take a look?